standard/classes.md

```diff
@@ -4223,8 +4227,8 @@ The `true` and `false` unary operators require pair-wise declaration. A compile-
 > public class IntVector
 > {
 >     public IntVector(int length) {...}
->     public int Length { ... }           // Read-only property
->     public int this[int index] { ... }  // Read-write indexer
+>     public int Length { get { /* ... */ } }                          // Read-only property
```

Comment:
It's slightly annoying that we can't just keep using ... for this. We may be able to add some "advanced configuration" to make it work... I'm fine with doing it like this for now, so long as we're consistent so we can find everywhere that needs it, and revert later on.

Reply:
I think we can here. I just failed to remove the comment delimiters when pasting the source back in from the post-extraction source